src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -248,6 +248,11 @@ private static bool ShouldOutputPlainText(bool isHost, bool? supportsVirtualTerm
                         break;
                 }
             }
+            else if (isHost)
```

Comment:
Can't we just change line 237 to default to PlainText?

Reply:
That's even better.